### PR TITLE
DATA: Add Refugee Migration

### PR DESCRIPTION
**Pull request by Spectralgo**
@ElbekDeveloper Please review the refugee Migration.

### CODE REVIEW

File: RefugeeLand.Core.Api/Migrations/20220511040903_AddRefugeeMigration.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Migrations/20220511040903_AddRefugeeMigration.cs:1]: File header @Spectralgo

File: RefugeeLand.Core.Api/Migrations/20220511041203_AddRefugeeMigrationAddMiddleName.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Migrations/20220511041203_AddRefugeeMigrationAddMiddleName.cs:1]: Header

File: RefugeeLand.Core.Api/Migrations/StorageBrokerModelSnapshot.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
+﻿// <auto-generated />
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Migrations/StorageBrokerModelSnapshot.cs:1]: Header

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: RefugeeLand.Core.Api/Migrations/20220511040903_AddRefugeeMigration.cs (hunk 1 above)
Comment (Spectralgo) [reply RefugeeLand.Core.Api/Migrations/20220511040903_AddRefugeeMigration.cs]: done

File: RefugeeLand.Core.Api/Migrations/20220511041203_AddRefugeeMigrationAddMiddleName.cs (hunk 2 above)
Comment (Spectralgo) [reply RefugeeLand.Core.Api/Migrations/20220511041203_AddRefugeeMigrationAddMiddleName.cs]: done

File: RefugeeLand.Core.Api/Migrations/StorageBrokerModelSnapshot.cs (hunk 3 above)
Comment (Spectralgo) [reply RefugeeLand.Core.Api/Migrations/StorageBrokerModelSnapshot.cs]: done

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**